app/admin/admin-charts-wrapper.tsx

```
@@ -0,0 +1,17 @@
+"use client";
+
+import dynamic from "next/dynamic";
+
+const AnalyticsCharts = dynamic(() => import("./analytics-charts"), {
+  ssr: false,
+  loading: () => (
+    <div className="flex items-center justify-center p-8">
+      <div className="animate-spin rounded-full h-8 w-8 border-b-2 border-primary"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)